administrator/language/en-GB/en-GB.mod_quickicon.ini

```diff
@@ -8,33 +8,50 @@ MOD_QUICKICON_ADD_NEW="New %s"
 MOD_QUICKICON_AMOUNT_THSD="THSD"
```

Comment:
What is this? Is it used?

Reply:
It is outdated, thanks for the hint.
The taget was to truncate numbers > 100000 but is not longer necessary